vsphere/test_vsphere.py

```diff
@@ -325,7 +325,8 @@ def test_mor_discovery(self):
         )
 
         # ...on VMs
-        self.assertMOR(instance, spec="vm", count=1)
+        # self.assertMOR(instance, spec="vm", count=1)
```

Comment:
remove?

Reply:
or revert? Checking for the count may be useful

Reply:
fixed.